frameworks/Nim/scorper/benchmark_config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "framework": "scorper",
+  "tests": [
+    {
+      "default": {
+        "json_url": "/json",
+        "plaintext_url": "/plaintext",
+        "db_url": "/db",
+        "query_url": "/queries?queries=",
+        "fortune_url": "/fortunes",
+        "update_url": "/updates?queries=",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "mysql",
```

Comment:
Is the database mysql or postgres? I see postgres stuff in code but you've asked to start mysql here.

Reply:
mysql or mariadb that use `mysql_native_password` , my dependency driver not implemented `caching_sha2_password_auth` , postgres stuff just copy from other nim project that not used.

Reply:
Please remove code here that isn't used, but I still see port 5432 in your bench file.

Reply:
ok, I compile and run `scorper_bench.nim` check browser /plaintext /json successfully responded, why it show `ERROR: Framework is not accepting requests from client machine`  both on locally and CI ?